int8 data1
bool data2
int32 data3
float32 data4
string data5
string data6

